synapse/lib/layer.py

```diff
@@ -2794,20 +2802,27 @@ async def fini():
 
             yield wind
 
-    async def getNodeEditOffset(self):
+    async def getEditIndx(self):
+        '''
+        Returns what will be the *next* (i.e. 1 past the last) nodeedit log index.
+        '''
         if not self.logedits:
             return 0
 
         return self.nodeeditlog.index()
 
-    async def getEditOffs(self, defv=None):
+    async def getEditOffs(self):
         '''
-        Return the offset of the last *recorded* log entry.
+        Return the offset of the last *recorded* log entry.  Returns -1 if nodeedit log is disabled or empty.
```

Comment:
Maybe this should return None rather than -1 when disabled to explode immediately rather than potentially allow math to be done on the return value.

Reply:
Math on None would be worse...